static/app/views/starfish/views/spanSummaryPage/sampleList/durationChart/index.tsx

```diff
@@ -41,20 +46,20 @@ function DurationChart({
     duration: number,
     compareToDuration: number
   ): {color: string; symbol: string} => {
-    if (isNearBaseline(duration, compareToDuration)) {
+    if (isNearAverage(duration, compareToDuration)) {
       return {
-        symbol: 'path://M 0 0 V -8 L 5 0 L 0 8 L -5 0 L 0 -8',
-        color: theme.gray300,
+        symbol: crossIconPath,
```

Comment:
still not sure about using the cross icon here since it can be hard to see sometimes.

from mocks: 
<img width="220" alt="image" src="https://github.com/getsentry/sentry/assets/18689448/3eb478b4-57b6-4f5f-be4a-3dcef08c017a">

@doralchan do you think we should pick another icon?

Reply:
Gonna merge this in for now, we can change in another PR